code/pizzabot.js

```diff
@@ -1,7 +1,62 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
-
-const pizzaPrice = 80
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaPrice = 80;
 
 //Put your Javscript code here:
+
+const orderPizza = () => {
+  alert(
+    `Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`
+  );
+  const orderName = prompt(
+    "Enter the name of the pizza you want to order today."
+  );
+
+  const validateOrderName = () => {
+    if (
+      orderName === vegetarian ||
+      orderName === hawaiian ||
+      orderName === pepperoni
+    ) {
+      //checks if orderName is true, if orderName false, go to the last else
+      return true;
+    } else {
+      alert("Choose a pizza that's on the menu");
+
+      return false;
+    }
+    /*if orderName false, should prompt again, or loop from start,
+     */
+  };
+
+  if (validateOrderName(orderName)) {
```

Comment:
Above if statement we can create new variable in which we will store result of `validateOrderName` function. Thanks to that, inside if statement we don't have to call our function, it will be called before and pass it's result into variable. Later on, we can use this variable in if statement.
There is a convention of how we should write boolean variables - they should always start with prefix 'is'. In our case, our variable could have a name `isOrderNameValid` or something like that.

Nevertheless, I'm happy you omitted `=== true` and just wrote 

` if (validateOrderName(orderName)) { ... }` 

rather than 

` if (validateOrderName(orderName) === true) { ... }`

Both these lines are the same, but first one is more clean :)